pypy2/python.pypy.nuspec

```diff
@@ -3,15 +3,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>python.pypy</id>
-    <version>7.1.0</version>
+    <version>7.3.1</version>
     <packageSourceUrl>https://github.com/refi64/chocolatey-pypy</packageSourceUrl>
     <owners>Ryan Gonzalez</owners>
     <title>PyPy</title>
     <authors>The PyPy Developers</authors>
     <projectUrl>http://www.pypy.org</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/refi64/chocolatey-pypy/7.1.0/pypy-icon.png</iconUrl>
-    <copyright>2003-2019 The PyPy Developers</copyright>
-    <licenseUrl>https://bitbucket.org/pypy/pypy/src/release-pypy2.7-v7.1.0/LICENSE</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/refi64/chocolatey-pypy/7.3.1/pypy-icon.png</iconUrl>
+    <copyright>2003-2020 The PyPy Developers</copyright>
+    <licenseUrl>https://bitbucket.org/pypy/pypy/src/release-pypy2.7-v7.3.1/LICENSE</licenseUrl>
```

Comment:
This URL is invalid.

Reply:
Looks like it has moved to https://foss.heptapod.net/pypy/pypy/blob/release-pypy2.7-v7.3.1/LICENSE

Reply:
Fixed.

Reply:
Thanks